tests/pytests/unit/state/test_state_compiler.py

```diff
@@ -1064,66 +960,11 @@ def test_mod_aggregate(minion_opts):
             ]
 
             # Ensure that the require requisite from the
-            # figlet state finds its way into this state
-            assert "require" in low_ret
+            # figlet state doesn't find its way into this state
+            assert "require" not in low_ret
 
             # Ensure pkgs were aggregated
-            assert low_ret["pkgs"] == ["figlet", "sl"]
-
-
-def test_mod_aggregate_order(minion_opts):
```

Comment:
And this one?

Reply:
This one is essentially a duplicate test and also is not correct.

Reply:
If I am remembering correctly (since I removed this half a year ago), because the test is not correct and is not how the ordering and mod aggregate works. If I am remembering correctly the requisites of the states stay the same and the network is just updated to determine the requisite ordering.